package/MDAnalysis/analysis/density.py

```diff
@@ -689,6 +691,23 @@ def density_from_Universe(universe, delta=1.0, atomselection='name OH2',
       (It should be noted that the `padding` keyword is not used when a user
       defined grid is assigned).
 
+    As detailed above, the :class:`Density` object returned contains a
+    physical density in units of Angstroem^{-3}. If you are interested in
```

Comment:
(As above)

Reply:
To make things easier in future efforts, I have now standardised the entire of density.py to Angstrom (matches what is declared in units.py).